### PR TITLE
Search Page Pagination and Styling

### LINKED ISSUE #48: Pagination
**Is your feature request related to a problem? Please describe.**
There is no pagination feature on the search page.

**Describe the solution you'd like**
Create pagination.

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1. Enable to use of pagination on the search page
2. Styled the search page 

## Purpose
To allow users to look through many pages of articles

## Approach
1. Created state values to house the total pages and page number values
2. Made functions to handle page up and down behaviors
3. Styled components for the styling

Closes #48

### CODE REVIEW

File: the-toad-tribune/src/SubApp.tsx [hunk 1]
```diff
@@ -146,9 +147,9 @@ const SubApp = () => {
   };
 
   useEffect(() => {
-    newsDataGrabber();
-    adsDataGrabber();
-    weatherDataGrabber();
+    // newsDataGrabber();
+    // adsDataGrabber();
+    // weatherDataGrabber();
```
Comment (Gjhnsn) [inline the-toad-tribune/src/SubApp.tsx:152]: This commented section needed still?

**Review by Gjhnsn — APPROVED** [review]
Great work!

File: the-toad-tribune/src/layout/SearchResults.tsx [hunk 2]
```diff
@@ -1,21 +1,72 @@
...
+  const [pageNumber, setPageNumber] = useState<number>(1);
+  const [totalPages, setTotalPages] = useState<number>(1);
+
+  const getTotalPages = () => {
+    if (searchResults.totalResults % 20 === 0) {
+      setTotalPages(searchResults.totalResults / 20);
+    } else {
+      setTotalPages(Math.floor(searchResults.totalResults / 20) + 1);
+    }
+  };
```
Comment (Lap343) [inline the-toad-tribune/src/layout/SearchResults.tsx:33]: Very nice

**Review by BlackCubes — APPROVED** [review]
NOICE!

File: the-toad-tribune/src/SubApp.tsx (hunk 1 above)
Comment (krferrell) [reply the-toad-tribune/src/SubApp.tsx:153]: Thanks! It is still needed.

**Review by Lap343 — APPROVED** [review]
Nice job cant wait to see it

**Review by gonzalespaulb — APPROVED** [review]

**Pull request merged**